config/types/schema.go

```diff
@@ -151,7 +149,7 @@ type PasswdGroup struct {
 type PasswdUser struct {
 	Create            *Usercreate        `json:"create,omitempty"`
 	Gecos             string             `json:"gecos,omitempty"`
-	Groups            []Group            `json:"groups,omitempty"`
```

Comment:
The `config/types/schema.go` file is autogenerated by schematyper. For some reason, schematyper renamed this type. I could drop this commit, but it probably means someone else has to go through this silly rename the next time they come along and regenerate this.

This rename is absolutely not necessary, it's just a side effect of the tool we're using.